include/odp/api/spec/schedule_types.h

```diff
@@ -165,6 +173,81 @@ typedef	struct odp_schedule_param_t {
 	uint32_t lock_count;
 } odp_schedule_param_t;
 
+/**
+ * Scheduler capabilities
+ */
+typedef struct odp_schedule_capability_t {
+	/** Maximum number of ordered locks per queue */
+	uint32_t max_ordered_locks;
+
+	/** Maximum number of scheduling groups */
+	uint32_t max_groups;
+
+	/** Number of scheduling priorities */
+	uint32_t max_prios;
+
+	/** Maximum number of scheduled (ODP_BLOCKING) queues of the default
+	 * size. */
+	uint32_t max_queues;
+
+	/** Maximum number of events a scheduled (ODP_BLOCKING) queue can store
+	 * simultaneously. The value of zero means that scheduled queues do not
+	 * have a size limit, but a single queue can store all available
+	 * events. */
+	uint32_t max_queue_size;
+
+	/** Maximum supported flows per queue.
+	 * Specifies the maximum number of flows per queue supported by the
+	 * implementation.  A value of 0 indicates flow aware mode is not
+	 * supported.
+	 */
+	uint32_t max_flows;
+
+	/** Lock-free (ODP_NONBLOCKING_LF) queues support.
+	 * The specification is the same as for the blocking implementation. */
+	odp_support_t lockfree_queues;
+
+	/** Wait-free (ODP_NONBLOCKING_WF) queues support.
+	 * The specification is the same as for the blocking implementation. */
+	odp_support_t waitfree_queues;
+
+} odp_schedule_capability_t;
+
+/**
+ * Schedule configuration
+ */
+typedef struct odp_schedule_config_t {
+	/** Maximum number of scheduled queues to be supported.
+	 *
+	 * @see odp_schedule_capability_t
+	 */
+	uint32_t num_queues;
+
+	/** Maximum number of events required to be stored simultaneously in
+	 * scheduled queue. This number must not exceed 'capa.queue.max_size'
+	 * required by the application.  A value of 0 configures default queue
+	 * size supported by the implementation.
+	 */
+	uint32_t queue_size;
```

Comment:
Broken sentence above:
"This number must not exceed 'capa.queue.max_size' required by the application."

Should be:
"This number must not exceed 'max_queue_size' capability. "

Reply:
ack

Reply:
Name of the capa field is "max_queue_size" (not max_size).